public/jsx/views/login.jsx

```diff
@@ -21,17 +22,20 @@ module.exports = React.createClass({
 
   componentDidMount: function() {
     var { router } = this.context;
-    UserStore.addSignInListener(this.onUserSignIn);
-    UserActions.signIn(router.getCurrentQuery().access_token);
+    var defer = userActions.signIn(router.getCurrentQuery().access_token);
+    defer(reduxConfig.default.dispatch);
```

Comment:
I found out a better way to do this. In the parent I can do:
````
<Login
  accessToken={state.accessToken}
  {...bindActionCreators(userActions, result.dispatch) }
/>
````
which then allows me to do:
````  
componentDidMount: function() {
  this.props.signIn(this.props.accessToken);
},
````